vendor/src/github.com/docker/distribution/manifest/schema2/manifest.go

```diff
@@ -18,7 +18,7 @@ const (
 	MediaTypeConfig = "application/vnd.docker.container.image.v1+json"
 
 	// MediaTypePluginConfig specifies the mediaType for plugin configuration.
-	MediaTypePluginConfig = "application/vnd.docker.plugin.v0+json"
+	MediaTypePluginConfig = "application/vnd.docker.plugin.image.v0+json"
```

Comment:
Even if every plugin tests is now failing because of this. I'll look into this shortly.